include/onnxruntime/core/graph/graph_viewer.h

```diff
@@ -42,6 +42,9 @@ class GraphViewer {
   /** Gets the Graph description. */
   const std::string& Description() const noexcept;
 
+  /** Gets the path of the owning model if any **/
+  const Path& ModelPath() const noexcept;
+
```

Comment:
Do we need to add this when it could be obtained via GetGraph().ModelPath()?

FWIW, once we added GetGraph we really diminished the value of a lot of things in this wrapper.

Reply:
Per offline discussion changed this to inline instead